pages.pt_BR/common/act.md

```diff
@@ -0,0 +1,28 @@
+# act
+
+> Executa GitHub Actions localmente utilizando Docker.
+> Mais informacoes: <https://github.com/nektos/act>.
```

Comment:
```suggestion
> Mais informações: <https://github.com/nektos/act>.
```

Reply:
I avoided using special characters to prevent strange messages in terminal. I'll fix it!